src/main/java/pl/hk/zadanie_25/TaskRepository.java

```diff
@@ -0,0 +1,12 @@
+package pl.hk.zadanie_25;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+
+public interface TaskRepository extends JpaRepository<Task, Long> {
+
+    @Query("SELECT t from Task t WHERE t.finished = true ")
+    List<Task> findAllTasksWhereFinishedIsTrue();
```

Comment:
Tutaj chcielibyśmy raczej użyć metody wygenerowanej przez Spring Data i dać to, czy zadanie ma być skończone, czy nie jako argument metody. Wtedy będziemy mogli wykorzystać to do obu przypadków.